firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/HAL/STM32/HAL.h

```diff
@@ -8,13 +8,14 @@
 
 #pragma once
 
+#include <Pufferfish/HAL/STM32/AnalogInput.h>
```

Comment:
It should be possible to just use relative imports for all these includes. They should all be included with quotes. For example, `#include "CRCChecker.h`